packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -64,7 +83,7 @@ module.exports = {
     // containing code from all our entry points, and the Webpack runtime.
     filename: 'static/js/bundle.js',
     // In development, we always serve from the root. This makes config easier.
-    publicPath: '/'
+    publicPath: publicPath 
```

Comment:
I don't understand why this change is necessary. This would break the development workflow because it would request files from CDN instead of the local server. Is this change in the development configuration intentional?